V2/tezex/src/components/wallet/Wallet.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from "react";
+import React, { useEffect } from "react";
```

Comment:
TS1259: Module '"/home/vishakh/dev/cryptonomic/TEZEX/V2/tezex/node_modules/@types/react/index"' can only be default-imported using the 'esModuleInterop' flag  index.d.ts(62, 1): This module is declared with 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.